package/MDAnalysis/lib/trans.pyx

```diff
@@ -0,0 +1,29 @@
+import numpy as np
+cimport cython
+cimport numpy as np
+
+
+@cython.boundscheck(False)
```

Comment:
you can turn this off globally in setup.py file. check my pytraj's one

https://github.com/Amber-MD/pytraj/blob/master/setup.py#L327-L347